app/feature_defaults_unittest.cc

```diff
@@ -213,7 +213,6 @@ TEST(FeatureDefaultsTest, DisabledFeatures) {
 #endif
       &media::kLiveCaption,
       &metrics::structured::kEnabledStructuredMetricsService,
-      &metrics::structured::kNearbyShareMetrics,
```

Comment:
Looks like this was meant to be in a separate commit (along with chromium_src/components/metrics/structured/structured_metrics_features.cc)?

Reply:
Removed here 6692557058cf95785c387853e04daa396c6aad4f and added back in separate commit here d12e0bf2e8d430b0f584446bb10d02cf4433e196